tests/cli/test_cmds_mark_for_deployment.py

```diff
@@ -138,52 +138,53 @@ def mock_status_instance_side_effect(service, instance):
+    assert mark_for_deployment.instances_deployed('cluster', 'service1', ['instance6'], 'somesha') == 0
+    assert mark_for_deployment.instances_deployed('cluster', 'service1', ['notaninstance'], 'somesha') == 0
+    assert mark_for_deployment.instances_deployed('cluster', 'service1', ['api_error'], 'somesha') == 0
+    assert mark_for_deployment.instances_deployed('cluster', 'service1', ['instance7'], 'somesha') == 1
+    assert mark_for_deployment.instances_deployed('cluster', 'service1', ['instance8'], 'somesha') == 1
 
 
-def are_instances_deployed_side_effect(cluster, service, instances, git_sha):
+def instances_deployed_side_effect(cluster, service, instances, git_sha):
     if instances == ['instance1', 'instance2']:
         return True
     return False
 
 
 @patch('paasta_tools.cli.cmds.mark_for_deployment.get_cluster_instance_map_for_service', autospec=True)
 @patch('paasta_tools.cli.cmds.mark_for_deployment._log', autospec=True)
-@patch('paasta_tools.cli.cmds.mark_for_deployment.are_instances_deployed', autospec=True)
+@patch('paasta_tools.cli.cmds.mark_for_deployment.instances_deployed', autospec=True)
 @patch('time.sleep', autospec=True)
-def test_wait_for_deployment(mock_sleep, mock_are_instances_deployed, mock__log,
+def test_wait_for_deployment(mock_sleep, mock_instances_deployed, mock__log,
                              mock_get_cluster_instance_map_for_service):
     mock_cluster_map = {'cluster1': {'instances': ['instance1', 'instance2', 'instance3']}}
     mock_get_cluster_instance_map_for_service.return_value = mock_cluster_map
 
-    mock_are_instances_deployed.side_effect = are_instances_deployed_side_effect
+    mock_instances_deployed.side_effect = instances_deployed_side_effect
     mock_sleep.side_effect = TimeoutError()
     with raises(TimeoutError):
         mark_for_deployment.wait_for_deployment('service', 'deploy_group_1', 'somesha', '/nail/soa', 1)
     mock_get_cluster_instance_map_for_service.assert_called_with('/nail/soa', 'service', 'deploy_group_1')
-    mock_are_instances_deployed.assert_called_with(cluster='cluster1',
-                                                   service='service',
-                                                   instances=mock_cluster_map['cluster1']['instances'],
-                                                   git_sha='somesha')
+    mock_instances_deployed.assert_called_with(cluster='cluster1',
+                                               service='service',
+                                               instances=mock_cluster_map['cluster1']['instances'],
+                                               git_sha='somesha')
 
     mock_cluster_map = {'cluster1': {'instances': ['instance1', 'instance2']},
                         'cluster2': {'instances': ['instance1', 'instance2']}}
     mock_get_cluster_instance_map_for_service.return_value = mock_cluster_map
-    mark_for_deployment.wait_for_deployment('service', 'deploy_group_1', 'somesha', '/nail/soa', 1)
```

Comment:
I think this line was important. Before this was the case that *shouldn't* raise a TimeoutError